style.css

```diff
@@ -0,0 +1,186 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-size: 16px;
+    background-color: #FE9D8D;
+    font-family: "Press Start 2P", sans-serif;
+}
+
+h1 {
+    color: #373C99;
+}
+
+.landingTopContainer {
+    display: flex;
+    justify-content: space-between;
+}
+
+.landingLeft {
+    width: 20%;
+}
+
+.landingTitleContainer {
+    justify-content: space-between;
+}
+
+.leaderSurfboardContainer {
+    display: flex;
+}
+
+.mainContainer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+#mainGameContainer {
+    margin: 5%;
+    width: 1000px;
+    height: 600px;
+    position: relative;
+    border-radius: 10px;
+    background: linear-gradient(#f9855d,#fdd288,#ffffff);
+    overflow: hidden;
+}
+
+.waves {
+    position: relative;
+    width: 100%;
+    height: 30vh;
+    margin-bottom:-7px;
+    min-height:100px;
+    max-height:150px;
+    z-index: -2;
+    
+}
+
+.sun {
+    width: 100px;
+    height: 100px;
+    position: absolute;
+    top: 370px;
+    right: 40px;
+    background-color: #f9cb5d;
+    border-radius: 50%;
+    z-index: 2;
+}
+
+.ocean {
+    height: 150px; /* change the height of the waves here */
+    width: 400%;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    overflow-x: hidden;
```

Comment:
I believe you should be able to remove this overflow rule - just because the parent has a fixed size with overflow hidden

Reply:
Removed as requested.